src/routes/Details.tsx

```diff
@@ -1,14 +1,208 @@
 import { useParams } from "react-router-dom";
+import { LineChart, AreaChart } from 'react-chartkick';
+import 'chartkick/chart.js'
```

Comment:
I think we imported this one twice. Maybe we can remove one and move the other to main.tsx since it's basically a static import